docs/modules/operation/pages/deep-dive/topology/enlinkd/introduction.adoc

```diff
@@ -22,8 +14,17 @@ They also allow the Kafka Producer to send TopologyMessage data.
 The following pages describe enlinkd and how to configure it, the supported discovery protocols, and the SNMP MIBs that must be exposed to gather links between nodes.
 You can view detailed information about a node's connections and supporting link data on the node detail page in the {page-component-title} web UI.
 
+[[ga-enlinkd-collectors]]
+== Enhanced linkd supported protocols
```

Comment:
```suggestion
== Enhanced linkd supported protocols

```

Reply:
commit e6ec71a2a49eb651d1dfd0327136b2dadf59ad12 (HEAD -> ar/jira/NMS-14909)
Author: Antonio <rssntn67@yahoo.it>
Date:   Tue Jan 10 10:20:42 2023 +0100

    Fixed Return space